src/Core/Migration/Migration1536233020OrderDelivery.php

```diff
@@ -24,7 +24,7 @@ public function update(Connection $connection): void
               `shipping_order_address_id` BINARY(16) NULL,
               `shipping_order_address_version_id` BINARY(16) NULL,
               `shipping_method_id` BINARY(16) NOT NULL,
-              `tracking_code` VARCHAR(200) COLLATE utf8mb4_unicode_ci NULL,
+              `tracking_codes` JSON NOT NULL,
```

Comment:
As we have released the first EarlyAccess version you now have to provide migrations for DB schema changes and can't just change old migrations.

So please provide a new Migration that does this schema change.

Also please add a note in the Upgrade.md file about your change.